test/telemetria_test.exs

```diff
@@ -97,7 +97,7 @@ defmodule Telemetria.Test do
         assert 0 == Example.annotated_1(nil)
         assert :forty_two == Example.annotated_1("42")
         assert 42 == Example.annotated_1(42)
-        Process.sleep(500)
+        Process.sleep(100)
```

Comment:
Documenting why this change happened in this pr my be helpful

Reply:
To pass the test suite faster.